tests/common/devices/fanout.py

```diff
@@ -173,8 +173,29 @@ def get_speed(self, interface_name):
         """
         return self.host.get_speed(interface_name)
 
+    def links_status_down(self, ports):
+        """Get interface status
+
+        Args:
+            ports (set): Interfaces on one fanout
+
+        Returns:
+            True: if all interfaces are down
+            False: if any interface is up
+        """
+        return self.host.links_status_down(ports)
+        
+     def links_status_up(self, ports):
```

Comment:
precommit error:
tests/common/devices/fanout.py:188:40: E999 IndentationError: unindent does not match any outer indentation level